pkg/crdconversion/egresspolicyconversion.go

```diff
@@ -0,0 +1,27 @@
+package crdconversion
```

Comment:
Maybe change the filename to `policy_egress.go` or `policy_egress_conversion.go`?

Reply:
I went with `policy_egress_conversion.go` and followed the same convention for all other files